Cards/Resource/Card_Debuff_healing.tres

```diff
@@ -0,0 +1,48 @@
+[gd_resource type="Resource" script_class="CardBase" load_steps=14 format=3 uid="uid://bek3vbtsf3x2l"]
+
+[ext_resource type="Script" path="res://Cards/Effects/EffectApplyStatus.gd" id="1_63tqs"]
+[ext_resource type="Script" path="res://Status/Debuffs/Debuff_healing.gd" id="2_7i6kf"]
+[ext_resource type="Script" path="res://Entity/Components/Stats/StatModifiers.gd" id="3_omw4j"]
+[ext_resource type="Script" path="res://Cards/Effects/EffectData.gd" id="4_hb64u"]
+[ext_resource type="Script" path="res://Cards/CardBase.gd" id="5_8c6ak"]
+[ext_resource type="Script" path="res://Cards/Targeting/TargetPlayer.gd" id="5_u0kut"]
+[ext_resource type="Script" path="res://Cards/EnergyData.gd" id="6_augqn"]
+
+[sub_resource type="Resource" id="Resource_d5x88"]
+script = ExtResource("3_omw4j")
+permanent_add = 0
+permanent_multiply = 1.0
+temporary_add = -1
+temporary_multiply = 1.0
+
+[sub_resource type="Resource" id="Resource_b0uet"]
+script = ExtResource("2_7i6kf")
+infinite_duration = false
+status_turn_duration = 3
+status_power = 1
+status_modifier_base_value = SubResource("Resource_d5x88")
+
+[sub_resource type="Resource" id="Resource_5niki"]
+script = ExtResource("1_63tqs")
+status_to_apply = SubResource("Resource_b0uet")
+
+[sub_resource type="Resource" id="Resource_7ilgy"]
+script = ExtResource("5_u0kut")
+
+[sub_resource type="Resource" id="Resource_8p0l4"]
+script = ExtResource("4_hb64u")
+effect = SubResource("Resource_5niki")
+value = 0
+targeting_function = SubResource("Resource_7ilgy")
+
+[sub_resource type="Resource" id="Resource_mrt6b"]
+script = ExtResource("6_augqn")
+energy_cost = 0
+
+[resource]
+script = ExtResource("5_8c6ak")
+application_type = 0
```

Comment:
0 is friendly only meaning we can't debuff the healing of enemies, this should be set to `All` or `Enemy only`